configureBuild.sh

```diff
@@ -179,6 +186,14 @@ setRepository() {
     suffix="openjdk/aarch32-port-jdk8u";
   elif [ "${BUILD_CONFIG[OPENJDK_CORE_VERSION]}" == "${JDK8_CORE_VERSION}" ] && [[ "${BUILD_CONFIG[OS_FULL_VERSION]}" == *"Alpine"* ]] && [[ "${BUILD_CONFIG[BUILD_VARIANT]}" == "${BUILD_VARIANT_TEMURIN}" ]]; then
     suffix="adoptium/alpine-jdk8u";
+  elif [[ "${BUILD_CONFIG[BUILD_VARIANT]}" == "${BUILD_VARIANT_HOTSPOT}" ]] && [ "${BUILD_CONFIG[OS_ARCHITECTURE]}" == "riscv64" ]; then
+    if [ "${BUILD_CONFIG[OPENJDK_CORE_VERSION]}" == "${JDK8_VERSION}" ] \
+      || [ "${BUILD_CONFIG[OPENJDK_CORE_VERSION]}" == "${JDK11_VERSION}" ] \
+      || [ "${BUILD_CONFIG[OPENJDK_CORE_VERSION]}" == "${JDK17_VERSION}" ]; then
+      suffix="openjdk/riscv-port-${BUILD_CONFIG[OPENJDK_CORE_VERSION]}"
```

Comment:
Similarly - this will need to be `OPENJDK_FOREST_NAME` instead of `OPENJDK_CORE_VERSION` similar to the lines underneath this to ensure it has the `u` suffix (Or adding an explicit `u` after the variable would also work)